app/src/main/java/de/westnordost/streetcomplete/quests/crossing_type/AddCrossingTypeForm.java

```diff
@@ -0,0 +1,42 @@
+package de.westnordost.streetcomplete.quests.crossing_type;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.quests.ImageListQuestAnswerFragment;
+
+public class AddCrossingTypeForm extends ImageListQuestAnswerFragment
+{
+    private final OsmItem[] TYPES = new OsmItem[] {
+            new OsmItem("traffic_signals", R.drawable.fire_hydrant_pillar, R.string.quest_crossing_type_signals),
```

Comment:
you probably want a different drawing for crossings, right now you use fire hydrant :)